src/Explorer/Controls/NotebookGallery/GalleryViewerComponent.tsx

```diff
@@ -161,8 +162,10 @@ export class GalleryViewerComponent extends React.Component<GalleryViewerCompone
       )
     );
 
-    tabs.push(this.createFavoritesTab(GalleryTab.Favorites, this.state.favoriteNotebooks));
-    tabs.push(this.createPublishedNotebooksTab(GalleryTab.Published, this.state.publishedNotebooks));
+    if (configContext.platform !== Platform.Hosted) {
+      tabs.push(this.createFavoritesTab(GalleryTab.Favorites, this.state.favoriteNotebooks));
```

Comment:
I am afraid this is incorrect.. We could have standalone DE but logged in user in which case we should show these tabs. 

If you click a notebook from gallery, you will see "Download to my notebooks" only if it is not stand alone gallery viewer. Please refer that code and use a similar check here.

Reply:
Modified it to use same logic as the download notebooks button